src/io.c

```diff
@@ -28,6 +28,7 @@ uint8_t io_event(uint8_t channel) {
                 THROW(EXCEPTION_IO_RESET);
             }
             /* fallthrough */
+            __attribute__((fallthrough));
```

Comment:
Without this annotation the compiler will throw warnings of unintended switch statement fallthrough.